core/pom.xml

```diff
@@ -91,6 +79,17 @@
 			</plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>test-jar</goal>
+						</goals>
```

Comment:
Create a test JAR so that we can depend on it in the `java-saml` module (to reuse test classes and resources).